tsconfig.json

```diff
@@ -5,7 +5,7 @@
     "baseUrl": ".",
     "esModuleInterop": true,
     "jsx": "preserve",
-    "lib": ["es2015", "es2016", "es2017"],
+    "lib": ["es2015", "es2016", "es2017", "dom"],
```

Comment:
keep this `dom` in memory - if you're going to work with fe

Reply:
I don't catch what do you mean by `fe`.